src/en/developer-terms.md

```diff
@@ -0,0 +1,58 @@
+Title: Writing charms using terms
+
+# Writing charms that use terms
+
+Many applications require the acceptance of Terms in order to be installed.
+Starting with Juju 2, charms can require a user accept its terms in order for
+it to be deployed.
+
+Because of the nature of terms, the user must have a Launchpad account, and will
+be prompted to login when attempting to deploy a term-enabled charm.
+
+# How it works
+
+When a user tries to deploy a term-enabled charm from the charm store, Juju will
+check if the terms have been agreed to. If they have, the charm will deploy as
+usual. If not, the user will be prompted to run the `juju agree` command in
+order to deploy.
+
+You can view which charm terms you've accepted by installing
+[charm-tools](./tools-charm-tools.html) and running `charm terms`.
+
+# Managing terms
+
+## Creating terms
+
+At the moment, terms can only be created by charm store administrators.
+Charm developers should send an email to juju@lists.ubuntu.com with:
```

Comment:
I'd recommend giving them a subject to use, that way we can add filters to catch these messages, also maybe we should ask them to fill out a form so they don't need to send their terms to the whole list